client/src/components/Home/index.tsx

```diff
@@ -213,6 +257,44 @@ export default class Home extends React.Component<any, IHomeState> {
     return (
       <div>
         <h1>Bot Analytics</h1>
+        
+      <Button
```

Comment:
Consider this (latest version):
![image](https://user-images.githubusercontent.com/13205761/27768343-ac134fa8-5f19-11e7-9a07-d010c0e14b2c.png)